pandas/core/series.py

```diff
@@ -1556,9 +1555,17 @@ def unique(self):
 
         Returns
         -------
-        ndarray or Categorical
-            The unique values returned as a NumPy array. In case of categorical
-            data type, returned as a Categorical.
+        ndarray or ExtensionArray
+            The unique values returned as a NumPy array. In case of an
+            extension-array backed Series, a new
+            :class:`~api.extensions.ExtensionArray` of that type with just
+            the unique values is returned. This includes
+
+            * Categorical
+            * Period
+            * Datetime with Timezone
```

Comment:
IntegerNA ?

Reply:
will edit.  note otherwise this is verbatim from 24024